scripts/install

```diff
@@ -123,53 +44,37 @@ ynh_script_progression --message="Configuring PHP-FPM..."
 
 # Create a dedicated PHP-FPM config
 ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint
-phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
-
-#=================================================
-# NGINX CONFIGURATION
-#=================================================
-ynh_script_progression --message="Configuring NGINX web server..."
 
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
+# Create a dedicated systemd config
+ynh_add_systemd_config
+
+yunohost service add $app --description="Responsive web-based XMPP client" --log="/var/log/$app/$app.log"
+
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding a configuration file..."
 
-ynh_add_config --template="../conf/db.example.inc.php" --destination="$final_path/config/db.inc.php"
+ynh_add_config --template="../conf/.env" --destination="$install_dir/.env"
 
-chmod 400 "$final_path/config/db.inc.php"
-chown $app:$app "$final_path/config/db.inc.php"
-
-#=================================================
-# SETUP SYSTEMD
-#=================================================
-ynh_script_progression --message="Configuring a systemd service..."
-
-# Create a dedicated systemd config
-ynh_add_systemd_config
+chmod 650 "$install_dir/.env"
+chown $app:$app "$install_dir/.env"
 
 #=================================================
 # BUILD MOVIM
 #=================================================
 ynh_script_progression --message="Building Movim..."
 
-ynh_exec_warn_less ynh_install_composer --phpversion="$phpversion" --workdir="$final_path"
-ynh_exec_warn_less ynh_composer_exec --phpversion="$phpversion" --workdir="$final_path" --commands="movim:migrate"
-ynh_exec_as $app php$phpversion $final_path/daemon.php config --username=$admin --password=$password --quiet
```

Comment:
uuh that thing seems to create the admin user, seems important ?

Reply:
I had trouble updating this app. I think this command is now obsolete but I will have to dig deeper to remember how to generate the admin (maybe a post install process)